lib/nmatrix/math.rb

```diff
@@ -984,13 +984,19 @@ def mean(dimen=0)
   #   sum(dimen) -> NMatrix
   #
   # Calculates the sum along the specified dimension.
+  
+  # call-seq:
+  #   cumsum() -> NMatrix
```

Comment:
This is not quite what I meant. The idea is to do something like:

```
# call-seq:
#   sum() -> NMatrix
#   cumsum() -> NMatrix
```

 instead. Make sense?